modules/spo-applying-profiles.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-applying-profiles_{context}"]
+= Applying {type} profiles to a pod
+
+Create a Pod to apply one of the created profiles.
+
+.Procedure
+
+ifdef::seccomp[]
+. Create a Pod object that defines a `securityContext`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
```

Comment:
Creating profiles referes to `example-profile` in `example-namespace`, here we refer to a pod without a namespace and refer to a profile in `my-namespace` called `profile1`. Could we make the examples follow the same profile so that the user could more or less copy-paste them?
(I know I like to follow examples as a user)

Reply:
Something like this?
```metadata:
  namespace: example-namespace
  name: example-pod

Reply:
Actually I'm confused - looks like we define example-namespace and then my-namespace in the "creating" page:
```
apiVersion: v1
kind: Pod
metadata:
  namespace: example-namespace <---
  name: example-pod
spec:
  securityContext:
    seccompProfile:
      type: Localhost
      localhostProfile: operator/my-namespace/profile1.json <---
  containers:
    - name: test-container
      image: nginx

Reply:
Right, this is what I tried to say. I think it would be better (for copy-pastability of the examples) to pick one example through the creating and applying page so that the user could just grab the examples and apply them to their cluster as they go.